index.html

```diff
@@ -1,83 +1,96 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <link rel="stylesheet" href="style.css">
-    <link rel="shortcut icon" href="./images/google.png" type="image/x-icon">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="shortcut icon" href="./images/google.png" type="image/x-icon" />
     <title>Google</title>
-</head>
-<body>
-
+  </head>
+  <body>
     <div class="userinfo">
```

Comment:
please make use of the proper html tag, use a section for this

Reply:
- [ ] It is always better to include an explanation, a "why", for each requested change. For example:

This is a well-explained request: *Why don't you use an article/section/aside/etc tag instead of a div, that would improve the HTML structure and make the page more accessible*

This request lacks a "why": *Why don't you use an article tag here instead of a div?*